### PR TITLE
feat(ecosystem): adds doc integration to directory

### PR DESCRIPTION
**Pull request by scefali**
This PR adds document integrations to the integration directory. Most of the content for descriptions came from Phillip but I made my own adjustments. This PR adds the following document integrations:

- Asayer
- Datadog
- Microsoft Teams
- RocketChat
- FullStory

Here's what it looks like in the directory:

Screen Shot 2020-04-06 at 10 42 04 AM

We also modified the `Learn More` button in the detailed view to have the external link icon:
Screen Shot 2020-04-06 at 10 42 12 AM

### CODE REVIEW

File: src/sentry/static/sentry/app/views/organizationIntegrations/integrationListDirectory.tsx [hunk 1]
```diff
@@ -187,39 +190,53 @@ export class IntegrationListDirectory extends AsyncComponent<
...
+    const out = this.getInstallValue(b) - this.getInstallValue(a);
+    return out;
+  };
 
   sortIntegrations(integrations: AppOrProviderOrPlugin[]) {
-    return integrations
-      .sort(this.sortByName)
-      .sort(this.sortByPopularity)
-      .sort(this.sortByInstalled);
+    return integrations.sort((a: AppOrProviderOrPlugin, b: AppOrProviderOrPlugin) => {
```
Comment (scefali) [inline src/sentry/static/sentry/app/views/organizationIntegrations/integrationListDirectory.tsx:226]: The old way we were sorting was incorrect and was actually not working in tests but worked just fine in the app

File: src/sentry/static/sentry/app/views/organizationIntegrations/integrationRow.tsx [hunk 2]
```diff
@@ -136,9 +150,15 @@ const PublishStatus = styled(({status, ...props}: PublishStatusProps) => (
 `;
 
 const CategoryTag = styled(
-  ({priority, category, ...p}: {category: string; priority: boolean; theme?: any}) => (
-    <div {...p}>{category}</div>
-  )
+  ({
+    priority: _priority,
```
Comment (scefali) [inline src/sentry/static/sentry/app/views/organizationIntegrations/integrationRow.tsx:154]: This change fixes an ES lint issue

File: src/sentry/static/sentry/app/views/organizationIntegrations/integrationListDirectory.tsx [hunk 3]
```diff
@@ -187,39 +190,53 @@ export class IntegrationListDirectory extends AsyncComponent<
...
     const weightA = this.getPopularityWeight(a);
     const weightB = this.getPopularityWeight(b);
     return weightB - weightA;
   };
 
-  sortByInstalled = (a: AppOrProviderOrPlugin, b: AppOrProviderOrPlugin) =>
-    this.getInstallValue(b) - this.getInstallValue(a);
+  sortByInstalled = (a: AppOrProviderOrPlugin, b: AppOrProviderOrPlugin) => {
+    const out = this.getInstallValue(b) - this.getInstallValue(a);
+    return out;
```
Comment (NisanthanNanthakumar) [inline src/sentry/static/sentry/app/views/organizationIntegrations/integrationListDirectory.tsx:222]: @scefali Nit: we don't need `out` as a variable. We can just return `this.getInstallValue(b) - this.getInstallValue(a);`

Comment (scefali) [reply src/sentry/static/sentry/app/views/organizationIntegrations/integrationListDirectory.tsx]: @NisanthanNanthakumar whoops...I meant to revert that change. Good catch.

**Review by NisanthanNanthakumar — APPROVED** [review]
Looks good to me!

**Pull request merged**